.travis.yml

```diff
@@ -15,8 +14,9 @@ install:
   - nvm install $NODE_VERSION
   - nvm use $NODE_VERSION
   - nvm alias default $NODE_VERSION
-  
+
   - npm ci
 
 script:
   - npm test
+  - npm run lint
```

Comment:
This is a total nitpick, but I think it's more standard to run the linter before the test suite. The reason being that the test suite is typically (although not in our case yet) orders of magnitude more time consuming than the linter, so it's best to see linter failures early and not clog the pipeline.